js/imgur.js

```diff
@@ -45,14 +77,23 @@ weechat.factory('imgur', ['$rootScope', function($rootScope) {
         fd.append("image", base64img); // Append the file
         fd.append("type", "base64"); // Set image type to base64
 
+        // Comment the next line if you don't want to upload your pics into a folder.
+        if(albEnabled == 'true') {
```

Comment:
If you use booleans as mentioned above, this would just be `if (albEnabled) { /* ... */ }`